lms/djangoapps/courseware/views/views.py

```diff
@@ -294,9 +294,9 @@ def yt_video_metadata(request):
             status_code = res.status_code
             if res.status_code == 200:
                 try:
-                    res = res.json()
-                    if res.get('items', []):
-                        response = res
+                    res_json = res.json()
```

Comment:
Nit: I would say that if we're renaming this we should rename it to something more explanatory like `response_json`.